.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules/
 .nx/
 lib/
-.env
\ No newline at end of file
+.env
+examples/node_modules/
+examples/lib/
```

Comment:
not needed I think. node_modules/ I think includes all instances of node_modules in the repo, but please let me know if I'm wrong